Vagrantfile

```diff
@@ -0,0 +1,24 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.box = "ubuntu/trusty64"
+  #config.vm.box_url = "https://atlas.hashicorp.com/ubuntu/trusty64"
+  config.vm.box_url = "https://atlas.hashicorp.com/ubuntu/boxes/trusty64/versions/20150928.0.0/providers/virtualbox.box"
```

Comment:
this is actually the default one https://vagrantcloud.com/ubuntu eventually links to it see https://vagrantcloud.com/ubuntu

but it wouldn't accept those as box_urls, we might be able to remove the box_url altogether to be honest